edgelet/iotedge/src/main.rs

```diff
@@ -247,7 +247,6 @@ fn run() -> Result<(), Error> {
         .subcommand(
             SubCommand::with_name("system")
                 .about("Manage system services for IoT Edge.")
-                .unset_setting(AppSettings::SubcommandRequiredElseHelp)
```

Comment:
Do we know why this was here to begin with?

Reply:
I assume Lee copied it from the subcommand above it, `init`, which did need this. At the time both `iotedge init` and `iotedge init import` were valid invocations.

Reply:
Ahh, that makes sense. Alright then.